src/components/team/Teammate.jsx

```diff
@@ -1,5 +1,37 @@
+import React from 'react';
```

Comment:
make sure you're not changing unrelated files within different issues (this is the slides issue but you're changing Teammate.jsx)